src/usr/local/www/system_groupmanager_addprivs.php

```diff
@@ -189,7 +189,7 @@ function get_root_priv_item_text() {
 $section->addInput(new Form_StaticText(
 	gettext('Privilege information'),
 	'<span class="help-block">'.
-	gettext('The following privileges effectively give root privilege to users in the group' .
+	gettext('The following privileges effectively give administrator-level privilege to users in the group' .
```

Comment:
s/privilege/access/ in this case to avoid awkward repetition of "privileges" in the sentence.